modules/pages/client/components/Contribute.component.js

```diff
@@ -59,6 +59,12 @@ export default function Contribute() {
                 </a>{' '}
                 to get an idea of what’s in the making and what you can do to
                 help out.
+                You could help Trustroots out with as little effort as{' '}
+                <a href="https://team.trustroots.org/Chat.html">
```

Comment:
One way to contact will probably work best — two is just more confusing?

Reply:
I haven't seen what is behind the hyperlink but the idea is that the Google form is behind it: that is what we created to collect the photos. The email is in case someone has questions afterwards or is a pro.

Reply:
The first part isn't my proposition. Only the second about donating photos. Behind "donating photo's" should be the link to the Google form we created for this.

Reply:
This is the text after someone checked for grammer:
You could also help Trustroots out with as little effort as donating
your photos to use for our website or create content. If you're a
(semi-)professional photographer or have any questions, please contact
us share@trustroots.org.

Reply:
What if we change the first sentence to: "One way to help us out is with little effort as donating your photos...."